include/ace/managers/game.h

```diff
@@ -26,40 +24,60 @@ typedef struct _tGameState {
 	tGameCb cbCreate;
 	tGameCb cbLoop;
 	tGameCb cbDestroy;
-	struct _tGameState *pPrev;
+	tGameCb cbSuspend;
+	tGameCb cbResume;
+	struct _tGameState *pStatePrev;
```

Comment:
It doesn't need to be named `pStatePrev`, since "state" component is deductible from field type and from context - what else could it be if not pointer to the state? It can stay, but I prefer as short names as possible if doesn't obfuscate the meaning.